python/test/unit/common/test_mpi.py

```diff
@@ -21,3 +27,55 @@ def test_mpi_comm_wrapper():
 
     assert isinstance(w1, MPI.Comm)
     assert isinstance(w2, MPI.Comm)
+
+
+def test_mpi_comm_wrapper_cppimport(tempdir):  # noqa: F811
+    """
+    Test MPICommWrapper <-> mpi4py.MPI.Comm conversion for code compiled with cppimport
+    """
+
+    cppimport = pytest.importorskip("cppimport")
+
+    @mpi_jit_decorator
+    def compile_module():
+        cpp_code_header = f"""
+        <%
+        setup_pybind11(cfg)
+        cfg['include_dirs'] += {dolfin_pc["include_dirs"] + [mpi4py.get_include()]}
+        cfg['compiler_args'] += {["-D" + dm for dm in dolfin_pc["define_macros"]]}
+        cfg['libraries'] += {dolfin_pc["libraries"]}
```

Comment:
This is probably overkill, as neither dolfin nor its dependencies are not needed here. Let me know if you want me to remove this.

Reply:
If it can be removed that would be good.